### PR TITLE
DO NOT MERGE: chore: use gts 2.0

### PR DESCRIPTION
**Pull request by alexander-fenster**
Seems it works!

I will not merge it now because @xiaozhenliu-gg5 is working on an async iterator feature and it will be a real bummer to merge her code with these changes. We'll merge it after that PR #199 is in.

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
Can we also remove prettier, and all the `eslint` libraries from this package.json, and the generated package.json(s)?

File: typescript/src/cli.ts [hunk 1]
```diff
@@ -14,15 +14,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as yargs from 'yargs';
-import { Generator } from './generator';
+import * as yargs from "yargs";
```
Comment (JustinBeckwith) [inline typescript/src/cli.ts]: Gah, that isn't right :(  Can you copy the integrated prettier config from other repos into the root?  I think for the next lib, we should do `npx gts init` to let it bootstrap these files.

**Comment by alexander-fenster** [comment]
Generated `package.json` is a different story, let's take care of those separately. They won't affect existing libraries anyway, only new ones, and I'll make this change when 2.0 is released (together with `engine` bump).  As for this `package.json`, neither prettier nor eslint is listed there.

File: typescript/src/cli.ts (hunk 1 above)
Comment (alexander-fenster) [reply typescript/src/cli.ts]: Done! Looks much less scary now!

**Pull request closed without merging**